src/projects/diff/group.html

```diff
@@ -178,52 +217,64 @@ <h2 hidden$="[[editable]]">[[displayTitle]]</h2>
           value: false,
           reflectToAttribute: true
         },
-        project: Object
+        project: Object,
+        _isAlreadyInitialized: Boolean
       },
+      observers: [
+        '_groupChanged(group, editable)'
+      ],
       attached: function() {
         if (!this.editable && !this.description) {
           this.hideDescription = true;
         }
-        if (this.editable) {
-          Sortable.create(this.$$('sortable-js'), {
-            group: 'diffs',
-            draggable: 'pull-request-diff-highlight',
-            ghostClass: 'hunk-ghost'
+        this.initSortable();
+      },
+      initSortable: function() {
+        if (this.editable && !this._isAlreadyInitialized) {
+          Sortable.create(this.$.hunks, {
+            group: 'hunks',
+            ghostClass: 'hunk-ghost',
+            handle: '.drag-handle',
+            animation: 500,
+            setData: function(dataTransfer, dragEl) {
+              // console.log(dragEl.hunk.img);
+              dataTransfer.setDragImage(dragEl.children[1].hunk.img, 50, 50);
```

Comment:
Hm this seems magic. If we change the structure it will likely break. Is there no cleaner way? Like `dragEl.$.hunk.hunk.img`?